src/measure_theory/pi.lean

```diff
@@ -252,6 +252,75 @@ lemma pi_hyperplane [∀ i, sigma_finite (μ i)] (i : ι) [has_no_atoms (μ i)]
 show measure.pi μ (eval i ⁻¹' {x}) = 0,
 from pi_eval_preimage_null _ (measure_singleton x)
 
+section intervals
+
+variables {μ} [Π i, partial_order (α i)] [∀ i, sigma_finite (μ i)] [∀ i, has_no_atoms (μ i)]
+
+lemma pi_Iio_ae_eq_pi_Iic {s : set ι} {f : Π i, α i} :
+  pi s (λ i, Iio (f i)) =ᵐ[measure.pi μ] pi s (λ i, Iic (f i)) :=
```

Comment:
Surely this can be factored through `(forall i, t i =ᵐ[\mu i] t' i) -> pi s t =ᵐ[measure.pi μ] pi s t'`? And the lemma with `forall` replaced by `exists` (or equivalently, dropped) is also true, right?

Reply:
The lemma with `exists` is not true: `[0, 1] × [0, 2]` is not the same as `[0, 1] × [0, 1]`.

Reply:
Good point, not sure what I was thinking there.